PSRT/PSRT.psd1

```diff
@@ -62,7 +62,8 @@ FunctionsToExport = @( 'Get-RTConfig',
                        'Get-RTTicket',
                        'Get-RTTicketEntry',
                        'New-RTSession',
-                       'Set-RTConfig'
+                       'Set-RTConfig',
+                       'Find-RTTicket'
```

Comment:
This let's PowerShell know which functions to export (otherwise we wouldn't see the function)